certification/internal/shell/operatorsdk.go

```diff
@@ -0,0 +1,45 @@
+package shell
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/errors"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/cli"
+)
+
+type OperatorSdkCLIEngine struct{}
+
+func (osce OperatorSdkCLIEngine) Scorecard(image string, opts cli.OperatorSdkScorecardOptions) (*cli.OperatorSdkScorecardReport, error) {
```

Comment:
nit: `s/osce/o/g` - I'd keep this struct reference simple here. Feel free to disregard.